### ISSUE TITLE
The hints stopped working with 0.4.0

### ISSUE THREAD

**Issue by hhyyrylainen** [Revolutionary-Games/Thrive#672]
With 0.4.0 the hints stopped working. They should probably be brought back.

**Comment by Untrustedlife** [comment]
Hints?

**Comment by hhyyrylainen** [comment]
The ones that zyad (I think) made. That told you like "collect more ammonia"

**Comment by Untrustedlife** [comment]
Ahh okay. I suppose it would be easy enough to add those back in with some kind of DisplayHint(“text”) method

**Comment by CI09** [comment]
Could you elaborate what the context hints precisely are? Like and example or something like that
Thanks

**Comment by hhyyrylainen** [comment]
I recall there being a few screenshots floating around that showed how this feature was. If I recall right there was a line of text in the microbe stage HUD somewhere that said stuff like "You need more glucose to stay alive" or stuff like that was very relevant to the player and showed some important information at that moment. That could also be extended to the editor to give tips on top of the suggested organelle feature.

So basically depending on the context and the situation the player is in, there would be short textual tips on what the player might want to do.

**Comment by hhyyrylainen** [comment]
With the improved tutorial etc. it kind of seems like this system doesn't have much need in the game right now.
